app.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const routes = require('./routes');
+const appDataSource = require('./models/appDataSource');
+const { globalErrorHandler } = require('./utils/error');
+
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(cors());
+app.use(express.json());
+app.use(morgan('combined'));
+app.use(routes);
+app.use(globalErrorHandler);
```

Comment:
routes에 존재하지 않은 endpoint로 들어오는 요청에 대해서 404 핸들링 할 수 있도록 아래와 같은 코드를 추가해서 핸들링해주세요. 추가로 이에 따라 heath check api 코드의 위치도 바뀌어야 합니다.

```suggestion
app.use(routes);

app.all('*', (req, res, next) => {
  const err = new Error(`Can't find ${req.originalUrl} on this server!`);

  err.statusCode = 404;

  next(err);
});

app.use(globalErrorHandler);
```